### ISSUE TITLE
Assimulo SVN checkout fails during build

### ISSUE THREAD

**Issue by marcusfuchs** [mbonvini/JModelicaDemo#3]
@mbonvini Thank you for sharing this great project! It was really helpful for me to get started with JModelica in Docker. 

When I tried to build the container, I got an error with the SVN checkout that seems to be caused by Assimulo and is probably the same problem as reported at http://www.jmodelica.org/27850#comment-6589

I worked around this changing 

```
RUN svn co https://svn.jmodelica.org/trunk JModelica.org
```
to 
```
RUN svn co https://svn.jmodelica.org/trunk JModelica.org; exit 0  # Notice the workaround `; exit 0`

# Workaround from http://www.jmodelica.org/27850#comment-6589
WORKDIR /home/docker/to_install/JModelica.org/external
RUN svn co https://svn.jmodelica.org/assimulo/trunk Assimulo
```

Just thought I'd report this in case it is useful to you and others. I could also create a PR for this, but it's a bit of a hack, so I wanted to raise the issue here first.

**Comment by mbonvini** [comment]
Hi @marcusfuchs nice to hear from you and thanks for posting here a solution!

Given that's this isn't the way it's supposed to work why don't you post the workaround in a separate branch? After that we can update the readme so people that are interested in your fix can actually use it.

If the problem won't be fixed and more people will find this problem we'll merge into `master`.

What do you think?

**Comment by marcusfuchs** [comment]
I agree, a separate branch can maybe help others and hopefully the SVN checkout process will be fixed, so that this workaround is no longer needed. I have uploaded my workaround to a fork at https://github.com/marcusfuchs/ModelicaInAction/tree/issue3_assimulo . Feel free to pull this branch into this repo or link to the fork. Or maybe it's enough to have the information here in this issue.
In any case, thanks again for sharing your project in the first place!

**Comment by sus536** [comment]
Hi ,

Thank you for this wonderful project. 
I have been trying to install it and I am getting the same error, and i think its because Jmodelica has now shifted to github. Is there a revised dockerfile?

Thank you

**Comment by marcusfuchs** [comment]
As far as I understand Modelon's announcement, only parts of the ecosystem have been moved to GitHub and JModelica itself is no longer available as open source. If this understanding is correct, I doubt whether it is possible to get a revised dockerfile to work. In any case, I personally have lost trust in the ecosystem and won't invest any more time into it. Sorry that I can't help any further.

**Comment by sus536** [comment]
Oh Okay, that makes sense.
Thank you very much.
